tracing_android_trace/README.md

```diff
@@ -32,7 +31,7 @@ This limitation is due to the underlying Android platform APIs.
 <img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread.">
 <figcaption>
 
-Tracing spans for [Vello](https://github.com/linebender/vello) shown in Android GPU Inspector
+Tracing spans for [vello](https://github.com/linebender/vello) shown in Android GPU Inspector
```

Comment:
If the idea behind lower case is to reference the crate then let's tick it.

```suggestion
Tracing spans for [`vello`](https://github.com/linebender/vello) shown in Android GPU Inspector
```

Reply:
```suggestion
Tracing spans for [Vello](https://github.com/linebender/vello) shown in Android GPU Inspector
```
This change was a copy-paste mistake; I wasn't intending to change this line.
Thanks for catching.